src/main/kotlin/org/rust/ide/intentions/WrapLambdaExprIntention.kt

```diff
@@ -32,9 +35,11 @@ class WrapLambdaExprIntention : PsiElementBaseIntentionAction() {
     override fun invoke(project: Project, editor: Editor, element: PsiElement) {
         val ctx = findContext(element) ?: return
         val lambdaBody = ctx.lambdaExpr.expr ?: return
-        val bodyStr = ctx.lambdaExpr.expr?.text ?: return
+        var bodyStr = ctx.lambdaExpr.expr?.text ?: return
+        bodyStr = "\n$bodyStr\n"
```

Comment:
Why do you put newlines here? There's no need for them, formatter should do the work when needed.